crates/c-api/include/wasmtime.h

```diff
@@ -223,26 +226,22 @@ extern "C" {
 /**
  * \brief Converts from the text format of WebAssembly to to the binary format.
  *
- * \param wat this it the input pointer with the WebAssembly Text Format inside of
- *   it. This will be parsed and converted to the binary format.
- * \param wat_len this it the length of `wat`, in bytes.
- * \param ret if the conversion is successful, this byte vector is filled in with
- *   the WebAssembly binary format.
+ * \param wat this it the input pointer with the WebAssembly Text Format inside
+ * of it. This will be parsed and converted to the binary format. \param wat_len
+ * this it the length of `wat`, in bytes. \param ret if the conversion is
+ * successful, this byte vector is filled in with the WebAssembly binary format.
```

Comment:
I think the doxygen bits may be corrupted here as well

Reply:
Fixed.